platform/android/export/gradle_export_util.h

```diff
@@ -142,4 +142,103 @@ Error _create_project_name_strings_files(const Ref<EditorExportPreset> &p_preset
+		if (hand_tracking_index == 1) {
+			manifest_xr_features += "    <uses-feature tools:node=\"replace\" android:name=\"oculus.software.handtracking\" android:required=\"false\" />\n";
+		} else if (hand_tracking_index == 2) {
+			manifest_xr_features += "    <uses-feature tools:node=\"replace\" android:name=\"oculus.software.handtracking\" android:required=\"true\" />\n";
+		}
+	}
+	return manifest_xr_features;
+}
+
+String _get_instrumentation_tag(const Ref<EditorExportPreset> &p_preset) {
+	String package_name = p_preset->get("package/unique_name");
+	String manifest_instrumentation_text = vformat(
+			"    <instrumentation\n"
+			"        tools:node=\"replace\"\n"
+			"        android:name=\".GodotInstrumentation\"\n"
+			"        android:icon=\"@mipmap/icon\"\n"
+			"        android:label=\"@string/godot_project_name_string\"\n"
+			"        android:targetPackage=\"%s\" />\n",
+			package_name);
+	return manifest_instrumentation_text;
+}
+
+String _get_plugins_tag(const String &plugins_names) {
+	if (!plugins_names.empty()) {
+		return vformat("    <meta-data tools:node=\"replace\" android:name=\"plugins\" android:value=\"%s\" />\n", plugins_names);
+	} else {
+		return "    <meta-data tools:node=\"remove\" android:name=\"plugins\" />\n";
+	}
+}
+
+String _get_activity_tag(const Ref<EditorExportPreset> &p_preset) {
+	bool uses_xr = (int)(p_preset->get("xr_features/xr_mode")) == 1;
+	String orientation = (int)(p_preset->get("screen/orientation")) == 1 ? "portrait" : "landscape";
+	String manifest_activity_text = vformat(
+			"        <activity android:name=\"com.godot.game.GodotApp\" "
+			"tools:replace=\"android:screenOrientation\" "
+			"android:screenOrientation=\"%s\">\n",
+			orientation);
+	if (uses_xr) {
+		String focus_awareness = bool_to_string(p_preset->get("xr_features/focus_awareness"));
+		manifest_activity_text += vformat("            <meta-data tools:node=\"replace\" android:name=\"com.oculus.vr.focusaware\" android:value=\"%s\" />\n", focus_awareness);
+	} else {
+		manifest_activity_text += "            <meta-data tools:node=\"remove\" android:name=\"com.oculus.vr.focusaware\" />\n";
+	}
+	manifest_activity_text += "        </activity>\n";
+	return manifest_activity_text;
+}
+
+String _get_application_tag(const Ref<EditorExportPreset> &p_preset, const String &plugins_names) {
+	bool uses_xr = (int)(p_preset->get("xr_features/xr_mode")) == 1;
+	String manifest_application_text =
+			"    <application android:label=\"@string/godot_project_name_string\"\n"
+			"        android:allowBackup=\"false\" tools:ignore=\"GoogleAppIndexingWarning\"\n"
+			"        android:icon=\"@mipmap/icon\">)\n";
+
+	manifest_application_text += _get_plugins_tag(plugins_names);
+	if (uses_xr) {
+		manifest_application_text += "        <meta-data tools:node=\"replace\" android:name=\"com.samsung.android.vr.application.mode\" android:value=\"vr_only\" />\n";
+	}
```

Comment:
Can you add a tag to remove the [`xr_mode_metadata_name`](https://github.com/godotengine/godot/blob/b7dc08fcf6117bc13b0b1210119e51b4c51f5086/platform/android/java/app/AndroidManifest.xml#L29) tag since it's only used with the legacy build.